### PR TITLE
Add C# Language Tabs to "Understanding Double Buffering" Guide

### PR DESCRIPTION
**Pull request by NidhishaPahade**
# Description

Added C# Top level and OOP code to the "Understanding Double Buffering" tutorial.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Documentation (update or new)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration.

- [x] Tested in latest Chrome
- [x] Tested in latest Firefox
- [x] npm run build
- [x] npm run preview

### If involving code

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### If modified config files

- [x] I have checked the following files for changes:
  - [ ] package.json
  - [ ] astro.config.mjs
  - [ ] netlify.toml
  - [ ] docker-compose.yml
  - [ ] custom.css

## Folders and Files Added/Modified

Please list the folders and files added/modified with this pull request.

- Added:
  - [x] NA
- Modified:
  - [x] src/content/docs/guides/Graphics/1-double-buffering.mdx

## Additional Notes

Screenshots of the code running succesfully:

Screenshot 2024-09-20 192324
Screenshot 2024-09-20 193150

### CODE REVIEW

**Comment by omckeon** [comment]
This guide actually didn't have any code for it, so will also need the C++ tabs.

**Comment by omckeon** [comment]
@NidhishaPahade I have now updated the files as I had suggested on Teams.

Please update the PR description to use the template from [this page](https://thoth-tech.netlify.app/products/splashkit/splashkit-tutorials/onboarding/04-pull-request-template/). With screenshots put at the end.

**Review by oliverexell-bruce — CHANGES_REQUESTED** [review]
Line 108, `window.CloseRequested("Double Buffering")` should be `window.CloseRequested`

**Review by oliverexell-bruce — APPROVED** [review]
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Documentation (update or new)

## How Has This Been Tested?

- [X] Tested in latest Chrome
- [X] Tested in latest Firefox
- [X] npm run build
- [X] npm run preview

## Folders and Files Added/Modified

Please list the folders and files added/modified with this pull request.

- Added:
  - [X] N/A
- Modified:
  - [X] src/content/docs/guides/Graphics/1-double-buffering.mdx

Looks good, great work!

File: src/content/docs/guides/Graphics/1-double-buffering.mdx [hunk 1]
```diff
@@ -41,3 +43,77 @@ So when drawing you need to keep two things in mind:
 
 1. Anything you draw will not appear unit you have refreshed the screen
```
Comment (kay-kaushik) [inline src/content/docs/guides/Graphics/1-double-buffering.mdx:44]: here until has a speling mistake, its written as unit

**Review by kay-kaushik — APPROVED** [review]
The code looks clean and is easy to understand. I know you just did the c# code but there is a spelling mistake which I pointed out, Fix that and it all looks good.

**Comment by omckeon** [comment]
This tutorial guide originally did not have any code so good work creating some example code. However, as this code doesn't relate to the content of the guide above (House drawing), and is missing the C++ tab, I will close this PR.
The closing of this PR won't affect your portfolio though.

**Pull request closed without merging**